include/core/parameters_lagrangian.h

```diff
@@ -167,8 +167,8 @@ namespace Parameters
       enum class InsertionMethod
       {
         file,
-        list,
```

Comment:
This was above because it is sorted alphabetically

Reply:
Not sure what yiu mean... Now it is sorted alphabetically

Reply:
Actually, I the one who put those in alphabetical order. Idk why I changed that in this PR... probably by accident when I removed the clear_and_add stuff.